examples/package.json

```diff
@@ -32,6 +32,8 @@
     "webpack": "^1.13.1"
   },
   "dependencies": {
+    "monaco-editor": "^0.9.0",
```

Comment:
It's installed if you run `npm install` in the parent directory otherwise it's not. 

IMHO the examples folder should show how to use the package in a simple project in a way that anyone can just copy the files and start hacking around without worrying about extra dependencies.

I know that it looks like a duplication but i think it's worth it